lib/src/entity.dart

```diff
@@ -1,5 +1,7 @@
 import 'package:flame/components.dart';
+import 'package:flame/game.dart';
 import 'package:flame_behaviors/flame_behaviors.dart';
+import 'package:flutter/material.dart';
```

Comment:
Why do we need the material import?

Reply:
Probably because of visibleForTesting? Unsure but is the only thing "out of normal"

Reply:
Foundation.dart provides that as well.